rafthttp/probing_status.go

```diff
@@ -44,11 +44,16 @@ func addPeerToProber(p probing.Prober, id string, us []string) {
 }
 
 func monitorProbingStatus(s probing.Status, id string) {
+	// set the first interval short to log error early.
+	interval := 5 * time.Second
```

Comment:
`statusErrorInterval = 5 * time.Second` next to `statusMonitoringInterval` declaration?